app/uk/gov/hmrc/pensionschemereturn/controllers/PsrVersionsController.scala

```diff
@@ -34,6 +35,30 @@ class PsrVersionsController @Inject()(cc: ControllerComponents, psrVersionsServi
     with Results
     with Logging {
 
+  def getVersionsForYears(
+    pstr: String,
+    startDates: Seq[String]
+  ): Action[AnyContent] = Action.async { implicit request =>
+    logger.debug(
+      s"Retrieving reporting versions for years- with pstr: $pstr, startDates: $startDates"
+    )
+    Future
+      .sequence(
```

Comment:
Future sequence doesn't provide any ordering, does this matter for the resulting JsArray?

Reply:
It doesn't matter for now, if needed we can order on the FE. But in practice I got the data back ordered with stubs, although I understand this is not guaranteed.